features/graph/api/src/test/java/org/opennms/netmgt/graph/api/generic/GenericGraphTest.java

```diff
@@ -86,4 +96,53 @@ public void shouldNotAllowNamespaceChangeAfterAddingElements(){
         assertThrowsException(IllegalStateException.class, () -> graphBuilder.properties(new MapBuilder<String, Object>()
                 .withProperty(GenericProperties.NAMESPACE, otherNamespace).build()));
     }
+
+    @Test
+    public void shouldConsiderSemanticZoomLevel() {
+        final String NAMESPACE = "dummy";
```

Comment:
shouldn't the variable called "namespace" (small letters) since it is not a constant?

Reply:
Technically, the correct naming would be `Namespace`. Anyways, I have no strong opinion about this, and just named it `namespace` as that seems to be the common use in OpenNMS